zuul.d/project-templates.yaml

```diff
@@ -459,6 +445,51 @@
     gate:
       queue: integrated
       jobs:
+        - ansible-test-network-integration-iosxr-netconf-python27-stable211:
```

Comment:
lets create a yaml anchor like we do here:

https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/project-templates.yaml#L140

To avoid a lot of copypasta.

then you'd do https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/project-templates.yaml#L163

We can call the anchor

ansible-collections-cisco-iosxr-jobs